### DISCUSSION TITLE
How to remove duplicate solution?

### DISCUSSION THREAD

**Discussion by vvithershins** [codewars/codewars.com#3081] [Q&A] [answered]
I clicked refactor on a completed solution without intending to change anything, just to view it again. Now I have two copies of the exact same solution under my completed solutions. How would I remove the copied version?

Thank you.

**Comment by Blind4Basics** [comment, marked as answer]
it's not possible.

When you want to "play with solutions", don't use the trainer, use the fork panel (=> click the fork button under the solution you're interested in. Yours or any other solution)
